app/src/main.rs

```diff
@@ -254,16 +254,37 @@ async fn main() -> anyhow::Result<()> {
         return Ok(());
     }
 
-    if let Commands::Assets { command, path } = &cli.command {
-        let path = ProjectPath::new_local(path.clone())?;
-        let manifest = load_manifest(&assets, &path).await?;
-
+    if let Commands::Assets { command } = &cli.command {
         match command {
-            AssetCommand::MigratePipelinesToml => {
+            AssetCommand::MigratePipelinesToml(opt) => {
+                let path = ProjectPath::new_local(opt.path.clone())?;
+                let manifest = load_manifest(&assets, &path).await?;
                 ambient_build::migrate::toml::process(&manifest, path.fs_path.unwrap())
                     .await
                     .context("Failed to migrate pipelines")?;
             }
+            AssetCommand::Import(opt) => match opt.path.extension() {
+                Some(ext) => {
+                    if ext == "wav" || ext == "mp3" || ext == "ogg" {
+                        let convert = opt.convert_audio;
+                        ambient_build::pipelines::import_audio(opt.path.clone(), convert)
+                            .context("failed to import audio")?;
+                    } else if ext == "fbx" || ext == "glb" || ext == "gltf" || ext == "obj" {
+                        let collider_from_model = opt.collider_from_model;
+                        ambient_build::pipelines::import_model(
+                            opt.path.clone(),
+                            collider_from_model,
+                        )
+                        .context("failed to import models")?;
+                    } else if ext == "jpg" || ext == "png" || ext == "gif" || ext == "webp" {
+                        // TODO: import textures API may change, so this is just a placeholder
+                        todo!();
+                    } else {
+                        eprintln!("Unsupported file type");
+                    }
+                }
+                None => eprintln!("Unknown file type"),
```

Comment:
I think it would be better to fail and return an error, rather than just printing text and exiting with SUCCESS.

Something like `=> anyhow::bail!("Unknown file type")` etc.

Silent errors that proceed anyways can be very frustrating since you easily miss a single line in the terminal if the rest of the command proceeds.